tests/acceptance/collection-test.js

```diff
@@ -60,3 +60,26 @@ test('collections do not share instances of proxies', function(assert) {
     }, /foo-bar-baz \[data-test-simple-list\] \[data-test-simple-list-item\]:eq\(0\)/);
   });
 });
+
+
+test('Collection works with PageObject definition', function(assert) {
+  let Foo = PageObject.extend({
+    scope: 'foo-page-object',
+
+    name() {
+      return 'This is Foo class';
+    }
+  });
+  let bar = PageObject.extend({
+    foos: collection(Object.assign({
```

Comment:
Shouldn't this just be `collection(Foo.scope('[data-test-simple-list-item]'))`?

Reply:
Yeap, that's cleaner